examples/src/demos/dev/ShaderMaterial.js

```diff
@@ -61,6 +59,7 @@ const data = [
   [img7, img14, disp15, -0.1],
   [img8, img15, disp13, 0.2],
   [img26, img25, disp8, -0.8],
+  [img17, img16, disp3, -0.8],
```

Comment:
Added back a final example to complete the layout (before it had a box missing).
<img width="869" alt="Screenshot 2020-06-01 at 12 28 11" src="https://user-images.githubusercontent.com/448410/83400594-6a177200-a403-11ea-96ae-07b7bb84a40d.png">